configs/sst_cs_plumbers-math.yaml

```diff
@@ -0,0 +1,18 @@
+document: feedback-pipeline-workload
+version: 1
+data:
+  name: Math:
+  description: Various math related libraries
+  maintainer: sst_cs_plumbers
+
+  packages:
+  - atlas
+  - atlas-devel
+    s390x:
```

Comment:
You're missing the `arch_packages` field above `s390x`.

Reply:
Thanks, should be fixed now